src/NBS.Appointments.Service/Models/SlotAvailabilityRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+using System;
+
+namespace NBS.Appointments.Service.Models
+{
+    public class SlotAvailabilityRequest
+    {
+        [JsonProperty("siteIdentifier")]
```

Comment:
I would be tempted just to call this "site" - more consistent and less cumbersome, although it is more accurate